frontend/src/app/selectors/news.js

```diff
@@ -6,16 +6,14 @@ import experimentSelector from './experiment';
 // from all available experiments and Test Pilot in general
 const newsUpdatesSelector = createSelector(
   store => store.news.updates,
-  store => store.browser.isDev,
```

Comment:
Hmm, trying to remember what use `isDev` was in here... Maybe it's made obsolete by having the component states demoed in Storybook.

Reply:
I think it was just for showing news updates with published dates in the future